src/main/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/RenderableDate.java

```diff
@@ -36,9 +36,15 @@ public RenderableDate(Date date, String format, String timezone) {
     @Override
     public String toString() {
         if (format != null) {
-            return format.equals("epoch") ?
-                String.valueOf(date.getTime()) :
-                formatCustom();
+            if (format.equals("epoch")) {
+                return String.valueOf(date.getTime());
+            }
+
+            if (format.equals("unix")) {
+                return String.valueOf(date.getTime() / 1000L);
```

Comment:
1000 is a magic number. Please extract to constant or use Duration to transform the time

Reply:
Added `private static final long DIVIDE_MILLISECONDS_TO_SECONDS = 1000L;`